test/e2e/resourceinterpreter_test.go

```diff
@@ -186,4 +186,21 @@ var _ = ginkgo.Describe("Resource interpreter webhook testing", func() {
 			})
 		})
 	})
+
+	ginkgo.Context("InterpreterOperation InterpretStatus testing", func() {
+		ginkgo.It("InterpretStatus testing", func() {
+			for _, cluster := range framework.ClusterNames() {
+				clusterDynamicClient := framework.GetClusterDynamicClient(cluster)
+				gomega.Expect(clusterDynamicClient).ShouldNot(gomega.BeNil())
+
+				memberWorkload := framework.GetWorkload(clusterDynamicClient, workloadNamespace, workloadName)
+				memberWorkload.Status.ReadyReplicas = *workload.Spec.Replicas
+				framework.UpdateWorkload(clusterDynamicClient, memberWorkload, cluster, "status")
+
+				klog.Infof("Waiting for workload(%s/%s) on cluster %s collecting correctly status", workloadNamespace, workloadName, cluster)
+				currentWorkLoad := framework.GetWorkload(clusterDynamicClient, workloadNamespace, workloadName)
+				gomega.Expect(currentWorkLoad.Status.ReadyReplicas).Should(gomega.Equal(*workload.Spec.Replicas))
+			}
```

Comment:
Shouldn't we check the object of `Work`?

Reply:
Is that we should check the whole WorkLoad.Status object ?